tests/primer/primer_tool.py

```diff
@@ -255,13 +259,30 @@ def _create_comment(
                 comment += "\n</details>\n\n"
 
         if comment == "":
-            comment = "🤖 According to the primer, this change has **no effect** on the checked open source code. 🤖🎉"
+            comment = (
+                "🤖 According to the primer, this change has **no effect** on the"
+                " checked open source code. 🤖🎉\n\n"
+            )
         else:
             comment = (
-                "🤖 **Effect of this PR on checked open source code:** 🤖\n\n" + comment
+                f"🤖 **Effect of this PR on checked open source code:** 🤖\n\n{comment}"
+            )
+        hash_information = (
+            f"*This comment was generated for commit {self.config.commit}*"
+        )
+        if len(comment) >= MAX_GITHUB_COMMENT_LENGTH:
```

Comment:
If the comment is under max, but is above max with the hash information this won't catch it. The logic needs to be a little different.

Reply:
Nice catch